apps/consumer-client/src/pages/examples/add-pcd.tsx

```diff
@@ -375,7 +378,10 @@ async function addSignatureProofPCD(messageToSign: string) {
 
 async function addIdentityPCD() {
   const newIdentity = await SemaphoreIdentityPCDPackage.prove({
-    identity: new Identity()
+    identity: {
+      argumentType: ArgumentTypeName.Identity,
+      value: new Identity()
```

Comment:
good catch. I think one more level of refactor to the prove function of the identity PCD would be good - that it takes an object argument `identity` that is *not* an instance of `Identity`. This would enable it to be used in the same way as the other PCDs. Right now I don't think it's possible to request a proof via the url scheme of an identity PCD (not that you would want to / should do that anyways).